File: 1.GettingStarted/printz.java
import java.util.*;

public class printz {
    public static void main(String args[])
    {
        int n = 5;

        for (int row = 1; row <= n ;row++)
        {
            for (int col = 1; col <= n ;col++)
            {
                if (row == 1 || row == n)
                System.out.print("*");
                else if (col+row == n)
                System.out.print("*");
                else
                System.out.print(" ");
            }
            System.out.println();
        }
    }

}

File: printz.py


n = 5
for row in range(1,n+1):
    for col in range(1,n+1):
        if(row == 1 or row == n ):
            print(col,end='')
        elif(row + col == n):
            print(col,end='')
        else:
            print(" ",end='')
    print("\n")